### ISSUE TITLE
Wrong link to reference article in Lucas-Kanade docs

### ISSUE THREAD

**Issue by ritvje** [pySTEPS/pysteps#185]
In the Lucas-Kanade function documentation (e.g. [here](https://pysteps.readthedocs.io/en/latest/generated/pysteps.motion.lucaskanade.dense_lucaskanade.html#pysteps.motion.lucaskanade.dense_lucaskanade)) there is a wrong link to the first referenced article. 
The reference is currently
```
Bouguet, J.-Y.: Pyramidal implementation of the affine Lucas Kanade feature tracker description of the algorithm, Intel Corp., 5, 4, https://doi.org/10.1109/HPDC.2004.1323531, 2001
```
But the doi link goes to some other article. I didn't find a DOI for the article, but at least in the [OpenCV documentation](https://docs.opencv.org/3.4/d0/de3/citelist.html#CITEREF_Bouguet00) that paper is linked to https://pdfs.semanticscholar.org/aa97/2b40c0f8e20b07e02d1fd320bc7ebadfdfc7.pdf.

**Comment by dnerini** [comment]
Hello @ritvje, thanks for pointing this out! I'll remove the wrong DOI from the references once #178 is merged into master.
